integration/bin/tachyon-master-yarn.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
+source "${SCRIPT_DIR}/common.sh"
+TACHYON_MASTER_JAVA_OPTS="${TACHYON_MASTER_JAVA_OPTS:-${TACHYON_JAVA_OPTS}}"
+
+mkdir -p "${TACHYON_LOGS_DIR}"
+
+"${JAVA}" -cp "${CLASSPATH}" \
+  ${TACHYON_MASTER_JAVA_OPTS} \
```

Comment:
Should this go last? Otherwise the user would not be able to modify the options below.